manager/watcher/rkt/rkt.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package container defines types for sub-container events and also
+// defines an interface for container operation handlers.
+package rkt
+
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/google/cadvisor/container/rkt"
+	"github.com/google/cadvisor/manager/watcher"
+
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+
+	rktapi "github.com/coreos/rkt/api/v1alpha"
+)
+
+type rktContainerWatcher struct {
+	// Signal for watcher thread to stop.
+	stopWatcher chan error
```

Comment:
I'm trying to keep it the same as the Raw watcher which does want to return an error value and the errors are collected by manager when it tries to stop all watchers.